yaml_specs/modulemd_packager_v2.yaml

```diff
@@ -3,15 +3,18 @@
 # for a module stream. It is always a proper subset of a `document: modulemd`
 # of the same `version`.
 document: modulemd-packager
+
 # Module metadata format version
 version: 2
 data:
     # A short summary describing the module, required
     summary: An example module
+
     # A verbose description of the module, required
     description: >-
         A module for the demonstration of the metadata format. Also,
         the obligatory lorem ipsum dolor sit amet goes right here.
+
```

Comment:
While adding blank lines for clarity, add one between `license` and `dependencies` below.